data/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 #message(STATUS "[JSKIMDEBUG] CMAKE_SOURCE_DIR = ${CMAKE_SOURCE_DIR}")
-file(GLOB DATAFILES ${CMAKE_SOURCE_DIR}/data/*)
-install(FILES ${DATAFILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/data)
+install(DIRECTORY ${CMAKE_SOURCE_DIR}/data/ValenciaMECq0q3 
```

Comment:
I think the plan is to hold the template files in `sbn_data` not in nusystematics. Please revert this change